### DISCUSSION TITLE
v7.2.3

### DISCUSSION THREAD

**Discussion by ssorallen** [tabwrangler/tabwrangler#446] [General]
Solve data errors faced by a number of Firefox users after the v7 major release.

Issues

* [#444] Firefox: An unexpected error occurred
* [#443] Tabs after session restore do not start their counter

<hr /><em>This discussion was created from the release <a href='https://github.com/tabwrangler/tabwrangler/releases/tag/v7.2.3'>v7.2.3</a>.</em>

**Comment by ivocosta82** [comment]
looks like chrome web store stopped being updated after version 6.9.0 , is there any other way of  getting this version of the extension?

**Reply by ssorallen** [reply]
@ivocosta82 Yes thanks for the reminder. I had been publishing only to Firefox as I made sure the refactoring of storage worked - Chrome has ~60k installs vs. Firefox's ~1k so Firefox has been the "beta testing" platform.

This release is stable, but I would like to re-verify on Chrome. This release necessarily must go to Chrome Web Store before [Chrome migrates to Manifest v3](https://developer.chrome.com/blog/resuming-the-transition-to-mv3/) in June 2024.

tl;dr v7 will go to Chrome when I can thoroughly test once more, and necessarily before June 2024.

**Reply by Slyger** [reply]
Will this chrome update have the hotkey for tab locking?

**Reply by ssorallen** [reply]
@Slyger Yes it will have the hotkey for tab locking.

**Reply by ssorallen** [reply]
[v7.3.1](https://github.com/tabwrangler/tabwrangler/releases/tag/v7.3.1) will be the release going out to Chrome. It required an update to the minimum supported Chrome version, going up to v88.

**Reply by ssorallen** [reply]
v7.3.3 is going out to Chrome users. Please comment in the [v7.3.3 discussion](https://github.com/tabwrangler/tabwrangler/discussions/459) if you run into any issues.
